js/src/modals/Verification/sms-store.js

```diff
@@ -67,6 +67,18 @@ export default class SMSVerificationStore extends VerificationStore {
     return hasReceivedCode(this.number, this.account, this.isTestnet);
   }
 
+  // SMS verification events don't contain the phone number, so we will have to
+  // send a new request every single time. See below.
+  @action checkIfAbleToRequest = () => {
```

Comment:
Hm, don't like `switch`, `setIfAbleToRequest` is ok. `isAbleToRequest` sounds like it's fully pure, so `check` seemed like a compromise. will change.